guides/v3.1.0/controllers/index.md

```diff
@@ -1,27 +1,31 @@
-## Controllers
+### What is a Controller?
 
-Controllers behave like a specialized type of Component that is rendered by
-the router when entering a Route.
+[Controllers](https://emberjs.com/api/ember/release/classes/Controller) are a routable object that is used to decorate the model returned by the router when entering a Route. A Controller is usually paired with an individual Route of the same name.
 
-The controller receives a single property from the Route – `model` – which is
-the return value of the Route's `model()` method.
```

Comment:
I think we need to mention these two lines above somewhere explicitly

Reply:
This is what I mean by routable :)